ethkey/cli/src/main.rs

```diff
@@ -170,7 +170,7 @@ fn main() {
 fn display(keypair: KeyPair, mode: DisplayMode) -> String {
 	match mode {
 		DisplayMode::KeyPair => format!("{}", keypair),
-		DisplayMode::Secret => format!("{:?}", keypair.secret()),
+		DisplayMode::Secret => format!("{}", keypair.secret().to_hex()),
 		DisplayMode::Public => format!("{:?}", keypair.public()),
```

Comment:
If we're not using the debug display for `Secret`, can we remove the debug display for `Public` and `Address` too? Since this is for displaying to a user these are a clear misuse of `Debug`.